grails-app/controllers/org/pih/warehouse/RoleInterceptor.groovy

```diff
@@ -20,6 +20,10 @@ class RoleInterceptor {
                                 'update', 'importData', 'receive', 'showRecordInventory', 'withdraw', 'cancel', 'change', 'toggle', 'exportAsCsv']
     def static changeControllers = ['createProductFromTemplate']
 
+    def static omittedControllersWithChangeActions = [
```

Comment:
I think it should be handled by the proper configuration of `openboxes.security.rbac.rules` in the `runtime.groovy`

Reply:
Thank you @awalkowiak